### DISCUSSION TITLE
mdfromat reformatting hyperlink URL field: bug or feature?

### DISCUSSION THREAD

**Discussion by rpdelaney** [executablebooks/meta#665] [Q&A] [answered]
This is part of mdformat doing the business on our github pull request template:

```diff
diff --git i/.github/PULL_REQUEST_TEMPLATE.md w/.github/PULL_REQUEST_TEMPLATE.md
index 04536a9..d486ea8 100644
--- i/.github/PULL_REQUEST_TEMPLATE.md
+++ w/.github/PULL_REQUEST_TEMPLATE.md
@@ -1,4 +1,4 @@
-# [Tracker story](<URL>)
+# [Tracker story](URL)

 <!--
     Insert the tracker story URL in the markdown link above
```

Why is it doing this? I'm inclined to call this a bug, but is there a motivation in the spec for removing the `<>`?

**Comment by hukkin** [comment, marked as answer]
Hey there!

This is a [documented feature](https://mdformat.readthedocs.io/en/stable/users/style.html#inline-links), not a bug.

> Why is it doing this?

Mdformat is an opinionated formatter. Many formatting decisions are made to achieve consistency and avoid having a document where the same syntax is expressed in varying styles.

In your example, the link renders to equal HTML regardless of whether the angle brackets are there or not, and mdformat's opinion is to not have them.
